Documentation/Index.rst

```diff
@@ -10,224 +16,252 @@ Welcome to the official TYPO3 Documentation
+            *  :ref:`Fluid is TYPO3’s templating engine. <t3coreapi:fluid>`
+               Fluid acts as the glue between your static HTML templates and the content you create in TYPO3’s backend.
 
+            * :ref:`Sitepackages <t3sitepackage:start>` allow you to bundle your Fluid templates and other site assets into a single, reusable extension that can be installed with a single click.
 
-.. _start-extdev:
 
-Develop custom extensions
-=========================
+   .. _start-extdev:
 
-:ref:`references` lists all relevant core manuals. One of these is
-:ref:`t3coreapi:start`. It contains detailed information for core and extension
-developers,
+   .. container:: col-md-6 pl-0 pr-3 py-3 m-0
 
-Getting started with extension development:
+      .. container:: card px-0 h-100
 
-*  :ref:`Introduction to extensions in TYPO3
-   <t3coreapi:extension-architecture-introduction>`
+         .. rst-class:: card-header
 
-*  Extension :ref:`t3coreapi:extension-files-locations`
+            .. rubric:: :ref:`Developing & using extensions <extensions>`
 
-*  :ref:`t3coreapi:extension-naming` and :ref:`t3coreapi:cgl`
+         .. container:: card-body
 
-More topics can be found in :ref:`t3coreapi:start`, for example:
+            Getting started with extension development:
 
-*  :ref:`QueryBuilder <t3coreapi:database-query-builder>` based on Doctrine
+            *  :ref:`Introduction to extensions in TYPO3
+               <t3coreapi:extension-architecture-introduction>`
 
-*  :ref:`t3coreapi:DependencyInjection`
-*  :ref:`t3coreapi:request-handling`
+            *  Extension :ref:`t3coreapi:extension-files-locations`
 
-If you are updating TYPO3 to the next major version, you may need to make
-changes in your custom extensions.
+            *  :ref:`t3coreapi:extension-naming` and :ref:`t3coreapi:cgl`
 
-The `Core changelog <https://docs.typo3.org/c/typo3/cms-core/master/en-us/>`__
-lists all relevant changes for each TYPO3 version since 7.
 
+   .. _start-configuration:
 
+   .. container:: col-md-6 pl-0 pr-3 py-3 m-0
 
-.. _start-configuration:
+      .. container:: card px-0 h-100
 
-Configure your TYPO3 installation
-=================================
+         .. rst-class:: card-header
 
-TYPO3 is well known for being extensive and easy to configure. The
-:ref:`t3coreapi:config-overview` in :ref:`TYPO3 Explained <t3coreapi:start>`
-gives you an overview of the various configuration options available to you.
+            .. rubric:: :ref:`Configuring TYPO3 <configuringTYPO3>`
 
-*  :ref:`Site Handling and Configuration <sitehandling-basics>` shows you how
-   to setup domains, languages, human-readable URLs and error pages.
+         .. container:: card-body
 
-*  :ref:`Backend User Management <t3start:user-management>` explains how you
-   setup backend users and grant them specific access to your installation of
-   TYPO3.
+            TYPO3 is well known for being extensive and easy to configure. The
```

Comment:
Hmm, I commented on this elsewhere. Is it being pulled in here automagically, so I don't need to comment on it here as well?